product-scenarios/11-Asynchronous-message-processing/11.1-Decoupling-interacting-parties-across-time-and-space/11.1.1-Decouple-parties-using-JMS/11.1.1.1-HTTP-ActiveMQ-HTTP/11.1.1.1.1-asynchronous-point-to-point-via-queue/src/test/java/org/wso2/carbon/ei/scenario/test/AsyncHttpToHttpViaActiveMqQueueTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.ei.scenario.test;
+
+import org.apache.http.HttpResponse;
+import org.awaitility.Awaitility;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.wso2.carbon.esb.scenario.test.common.ScenarioConstants;
+import org.wso2.carbon.esb.scenario.test.common.ScenarioTestBase;
+import org.wso2.carbon.esb.scenario.test.common.elasticsearch.ElasticSearchClient;
+import org.wso2.carbon.esb.scenario.test.common.http.HttpConstants;
+import org.wso2.carbon.esb.scenario.test.common.http.RESTClient;
+
+import java.io.IOException;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Test class to test asynchronous point to point invocation via message queue with ActiveMQ
+ */
+public class AsyncHttpToHttpViaActiveMqQueueTest extends ScenarioTestBase {
+
+
```

Comment:
Shall we remove this unnecessary new line?

Reply:
Updated